pkg/codegen/docs/templates/properties.tmpl

```diff
@@ -11,8 +11,8 @@
         <span class="property-type">{{- if eq .Type.Link "#" "" -}}{{- htmlSafe .Type.Name -}}{{- else -}}{{ template "linkify" .Type }}{{- end -}}</span>
     </dt>
     <dd>
-        {{- print "{{% md %}}" -}}{{- htmlSafe .Comment -}}{{- print "{{% /md %}}" -}}
-        {{- if .DeprecationMessage -}}<p class="property-message">Deprecated: {{ print "{{% md %}}" -}}{{- .DeprecationMessage -}}{{- print "{{% /md %}}" -}}</p>{{- end -}}
+        {{- if .DeprecationMessage -}}<p class="property-message">Deprecated: {{ print "{{% md %}}" -}}{{- .DeprecationMessage -}}{{- print "{{% /md %}}" -}}</p>
```

Comment:
@cnunciato just to be on the same page, should we do render this information as an "either or" type of thing or render both, but make sure to strip off the deprecation from the comment itself? I guess I thought we wanted to do the latter? Looking at the issue itself, I guess it is not clear to me which one you would recommend here?

Reply:
I was hoping we'd be able to apply some sort of a regex to strip out the duplicate content somehow, yeah. Having duplicates seems slightly better to me than not having any description at all. Thoughts?

Reply:
Yeah especially in some cases the deprecation message may not actually be duplicated in the property comment, in which case, we would want to show both. For example, here's one such case where having both is necessary:

This is on https://www.pulumi.com/docs/reference/pkg/aws/ec2/instance/.
![Screenshot_2020-04-13 Instance](https://user-images.githubusercontent.com/1466314/79152412-fe107a00-7d80-11ea-819f-97e3a07b5ce9.png)

I do think it would be good to look at both cases to understand why the deprecation message is duplicated in the comment in one, but not the other.

Reply:
I'll keep looking into this, thanks for the feedback @praneetloke and @cnunciato .

Reply:
Great! Thank you, @jkisk !